catroid/src/main/java/org/catrobat/catroid/utils/VibrationUtil.java

```diff
@@ -124,52 +112,41 @@ public static void reset() {
 		setTimeToVibrate(0.0);
 	}
 
-	public static void setContext(Context stageContext) {
-		context = stageContext;
+	public static void setVibrator(Vibrator vibrator) {
+		vibration = vibrator;
```

Comment:
The member function should be called vibrator as well, in accordance with the setter name,